### PR TITLE
PBD-3182 Support custom default branch naming

### PR DESCRIPTION
**Pull request by fruiti-lewis**

### CODE REVIEW

File: build.sbt [hunk 1]
```diff
@@ -4,10 +4,9 @@ import sbt._
 val appName = "init-repository"
 
 lazy val library = Project(appName, file("."))
-  .enablePlugins(SbtAutoBuildPlugin, SbtGitVersioning, SbtArtifactory)
```
Comment (chotai) [inline build.sbt:7]: These three are auto enabled now so you can remove this line.

File: project/plugins.sbt [hunk 2]
```diff
@@ -1,12 +1,8 @@
...
-  Resolver.ivyStylePatterns)
+resolvers += MavenRepository("HMRC-open-artefacts-maven2", "https://open.artefacts.tax.service.gov.uk/maven2")
+resolvers += Resolver.url("HMRC-open-artefacts-ivy", url("https://open.artefacts.tax.service.gov.uk/ivy2"))(Resolver.ivyStylePatterns)
 
-resolvers += "HMRC Releases" at "https://dl.bintray.com/hmrc/releases"
-
-addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "1.13.0")
+addSbtPlugin("uk.gov.hmrc" % "sbt-auto-build" % "3.0.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-git-versioning" % "1.15.0")
```
Comment (chotai) [inline project/plugins.sbt:6]: This is now transitively pulled in so can be deleted from here

**Review by chotai — APPROVED** [review]
Change itself looks good. Few sbt-auto-build clean ups that can now be done

File: build.sbt [hunk 3]
```diff
@@ -1,13 +1,13 @@
 import sbtassembly.AssemblyKeys.assembly
 import sbt._
 
+scalaVersion := "2.11.12"
```
Comment (fruiti-lewis) [inline build.sbt:4]: Not sure if this is actually required, once I removed the transitive plugins and sbt-auto-build it only seemed to compile 2.10 - so added this

**Review by chotai — APPROVED** [review]
LGTM

**Pull request merged**